Update docs for webhook secret and other fixes

sm43 opened the pull request:
this updates docs for configuring webhook secret for
valiation for providers and fixes other broken links and
spells/grammer.

Signed-off-by: Shivam Mukhade <smukhade@redhat.com>

<!-- 🎉🎉🎉 Thank you for the PR!!! 🎉🎉🎉 -->

# Changes

<!-- Describe your changes here- ideally you can get that description straight from
your descriptive commit message(s)! -->

# Submitter Checklist

- [x] ♽  Run `make test lint` before submitting a PR (ie: with [pre-commit](https://pipelinesascode.com/dev/tools), no need to waste CPU cycle on CI
- [x] 📖 If you are adding a user facing feature or make a change of the behavior, please verify that you have documented it
- [x] 🧪 100% coverage is not a target but most of the time we would rather have a unit test if you make a code change.
- [x] 🎁 If that's something that is possible to do please ensure to check if we can add a e2e test.
- [x] 🔎 If there is a flakiness in the CI tests then don't *necessary* ignore it, better get the flakyness fixed before merging or if that's not possible there is a good reason to bypass it. (token rate limitation may be a good reason to skip).

chmouel commented on docs/content/_index.md:
```diff
@@ -48,20 +48,20 @@ of the execution when triggered by a `Pull Request` or a `Push`.
 ## Getting Started
 
-The easiest way to get started is to use the `tkn pac` CLI and its [bootstrap](/docs/guide/cli/#commands) command.
+The easiest way to get started is to use the `tkn pac` CLI and its [bootstrap](./docs/guide/cli.md#install) command.
```
those links are not going to work on hugo :-D you can see the preview here : 

https://preview-pipelines-as-code-ci.apps.paac.devcluster.openshift.com/docs/eaa14f68dd791478a036cd54675f70df04cbc096/docs/guide/cli.md#install

:-D 

the best way to develop docs and preview the changes is to do : 

```
cd docs
hugo server
```

go to https://localhost:1313 and your change should automatically be displayed in browser live....

(i'll add it to the dev docs)

sm43 replied:
ah ok were they working before ?
and i broke them 😅 
I was getting error in goland for broken link so fixed it..

chmouel replied:
yeah they are working but only via hugo generated :)

chmouel commented on docs/content/_index.md:
```diff
@@ -48,20 +48,20 @@ of the execution when triggered by a `Pull Request` or a `Push`.
 
 and follow the questions to get Pipelines as Code installed on your cluster.
-It will then help you to create create a GitHub Application to connect your repositories to Pipelines as Code.
+It will then help you to create a GitHub Application to connect your repositories to Pipelines as Code.
```
```suggestion
It will then help you create a GitHub Application to connect your repositories to Pipelines as Code.
```

chmouel commented on docs/content/_index.md:
```diff
@@ -48,20 +48,20 @@ of the execution when triggered by a `Pull Request` or a `Push`.
 If you are in a source code project, it will immediately ask you if you want to have a sample `PipelineRun` for `Pipelines as Code`
 
-## Walkthrough video
+## Walk-through video
```
spell checker say Walk-through but google says Walkthrough is an `adjective` :D 

![image](https://user-images.githubusercontent.com/98980/162702890-ee926815-c183-4217-b41a-e158d0cef357.png)

chmouel commented on docs/content/_index.md:
```diff
@@ -70,8 +70,8 @@ This 10-minute video will guide you through the `tkn-pac bootstrap` flow :
 
 For more details on the different installation methods please follow [the
-installation document](/docs/install/overview) detailing the Pipelines as Code
+installation document](./docs/install/overview) detailing the Pipelines as Code
```
yeah same as before that will be broken with hugo

chmouel commented on docs/content/_index.md:
```diff
@@ -70,8 +70,8 @@ This 10-minute video will guide you through the `tkn-pac bootstrap` flow :
 
 If you need to use `Pipelines as Code` and author `PipelineRuns` you can follow
-the [usage guide](/docs/guide)
+the [usage guide](./docs/guide)
```
ditto

chmouel commented on docs/content/docs/install/bitbucket_cloud.md:
```diff
@@ -22,17 +22,17 @@ check these boxes to add the permissions to the token :
 
 {{< hint info >}}
-[Refer to this screenshot](/images/bitbucket-cloud-create-secrete.png) to verify you have properly configured the app password.
+[Refer to this screenshot](./../../../static/images/bitbucket-cloud-create-secrete.png) to verify you have properly configured the app password.
```
same same :D

chmouel reviewed (changes requested):
the links are going to be broken on pipelinesascode.com

sm43 commented:
let me test it locally 😄

chmouel commented:
you can see it on the preview url too :D https://preview-pipelines-as-code-ci.apps.paac.devcluster.openshift.com/docs/4b641ba416ace0177584079f184c5805c24e169c/

sm43 commented:
fixed it .. ptal :)

chmouel commented on docs/content/dev/release-process.md:
```diff
@@ -40,13 +40,13 @@ git tag 1.2.3
 # Issues you may see
 
-* Sometime there may be some issues with system or others. If you need to rekick the release process you need to :
+* Sometimes there may be some issues with system or others. If you need to re-kick the release process you need to :
```
```suggestion
* Sometimes, there may be some issues with system or others. If you need to re-kick the release process you need to :
```

Commit:
Update docs for webhook secret and other fixes

this updates docs for configuring webhook secret for
valiation for providers and fixes other broken links and
spells/grammer.

Signed-off-by: Shivam Mukhade <smukhade@redhat.com>
